### PR TITLE
CFY-2449 internal value should start with '_'

### PR DESCRIPTION
**Pull request by adamlavie**

### CODE REVIEW

File: cloudify_rest_client/blueprints.py [hunk 1]
```diff
@@ -99,14 +99,18 @@ def _upload(self, archive_location,
         return self.api.put(uri, params=query_params, data=data,
                             expected_status_code=201)
 
-    def list(self, _include=None):
+    def list(self, _include=None, **kwargs):
         """
         Returns a list of currently stored blueprints.
 
         :param _include: List of fields to include in response.
```
Comment (adamlavie) [inline cloudify_rest_client/blueprints.py]: check that the @ is not made up

File: cloudify_rest_client/node_instances.py [hunk 2]
```diff
@@ -146,7 +149,11 @@ def list(self, deployment_id=None, node_name=None, _include=None):
         if deployment_id:
             params['deployment_id'] = deployment_id
         if node_name:
-            params['node_name'] = node_name
```
Comment (adamlavie) [inline cloudify_rest_client/node_instances.py]: remove comment once resolved

File: cloudify_rest_client/nodes.py [hunk 3]
```diff
@@ -155,7 +157,9 @@ def list(self, deployment_id=None, node_id=None, _include=None):
         if deployment_id:
             params['deployment_id'] = deployment_id
         if node_id:
-            params['node_id'] = node_id
```
Comment (adamlavie) [inline cloudify_rest_client/nodes.py]: remove comment once resolved

**Comment by barakm** [comment]
@adamlavie what is the status here?

**Comment by adamlavie** [comment]
related to the current feature I'm working on (rest list filtering)


________________________________

File: cloudify_rest_client/nodes.py [hunk 4]
```diff
@@ -148,6 +147,8 @@ def list(self, deployment_id=None, node_id=None, _include=None):
         :param deployment_id: The deployment's id to list nodes for.
         :param node_id: If provided, returns only the requested node.
```
Comment (adamlavie) [inline cloudify_rest_client/nodes.py]: add "deprecated, use id"

File: cloudify_rest_client/node_instances.py [hunk 5]
```diff
@@ -129,24 +128,35 @@ def update(self,
...
+             _include=None, **kwargs):
         """
         Returns a list of node instances which belong to the deployment
         identified by the provided deployment id.
 
         :param deployment_id: Optional deployment id to list node instances
                               for.
-        :param node_name: Optional node id to only fetch node instances with
-                          this name
+        :param node_name: Optional node name to only fetch node instances with
```
Comment (adamlavie) [inline cloudify_rest_client/node_instances.py]: node_name as positional argument will be deprecated

**Pull request merged**